Command/BreakpointCommand.php

```diff
@@ -80,5 +80,6 @@ protected function execute(InputInterface $input, OutputInterface $output)
             // Toggle the breakpoint.
             $this->getManager()->toggleBreakpoint('default', $version);
         }
+        return self::CODE_SUCCESS;
```

Comment:
Please add a new line before the `return` to be consistent with the others Command script. 👍🏻